packages/svelte-vega/package.json

```diff
@@ -54,17 +54,17 @@
 		"eslint": "^8.39.0",
 		"eslint-config-prettier": "^8.8.0",
 		"eslint-plugin-svelte3": "^4.0.0",
-		"prettier": "^2.8.8",
-		"prettier-plugin-svelte": "^2.10.0",
+		"prettier": "^3.0.0",
+		"prettier-plugin-svelte": "^3.0.0",
 		"publint": "^0.1.11",
-		"svelte": "^3.59.2",
+		"svelte": "^3.59.2 || ^4.0.0",
```

Comment:
This can just be svelte 4 since that’s what we will be testing against. I don’t think we get any benefit from supporting svelte 3 here.

Reply:
```suggestion
		"svelte": "^4.0.0",
```